README.md

```diff
@@ -1,73 +1,31 @@
+hipSOLVER requires either cuSOLVER or rocSOLVER + rocBLAS to be installed on the system. Once these are installed, the following commands will build hipSOLVER and install to `/opt/rocm/hipsolver`:
 
-* Ubuntu: `sudo apt install libboost-program-options-dev`
-* Fedora: `sudo dnf install boost-program-options`
+    cd hipSOLVER
+    ./install.sh -i
 
-Unfortunately, googletest and lapack are not as easy to install. Many distros do not provide a googletest package with pre-compiled libraries, and the lapack packages do not have the necessary cmake config files for cmake to configure linking the cblas library. hipSOLVER provide a cmake script that builds the above dependencies from source. This is an optional step; users can provide their own builds of these dependencies and help cmake find them by setting the `CMAKE_PREFIX_PATH` definition. The following is a sequence of steps to build dependencies and install them to the cmake default, /usr/local.
+Once installed, hipSOLVER can be used just like any other library with a C API. The header file will need to be included in the user code, and the hipSOLVER library will become a link-time and run-time dependency for the user application.
 
-```sh
-mkdir -p <HIPSOLVER_BUILD_DIR_PATH>/release/deps
-cd <HIPSOLVER_BUILD_DIR_PATH>/release/deps
-cmake -DBUILD_BOOST=OFF <HIPSOLVER_SOURCE_PATH>/deps   #assuming boost is installed through package manager as above
-make -j$(nproc) install
-```
-#### Library and clients
-Once dependencies are available on the system, it is possible to configure the clients to build. This requires a few extra cmake flags to the library's cmake configure script. If the dependencies are not installed into system defaults (like /usr/local), you should pass the `CMAKE_PREFIX_PATH` to cmake to help find them.
-* `-DCMAKE_PREFIX_PATH="<semicolon separated paths>"`
-
-```sh
-CXX=/opt/rocm/bin/hcc cmake -DBUILD_CLIENTS_TESTS=ON -DBUILD_CLIENTS_BENCHMARKS=ON [HIPSOLVER_SOURCE]
-make -j$(nproc)
-sudo make install   # sudo required if installing into system directory such as /opt/rocm
-```
+For more information on building and installing hipSOLVER, see the [hipSOLVER install guide](https://hipsolver.readthedocs.io/en/latest/userguide_install.html)
 
 ## Using the hipSOLVER Interface
-The hipSOLVER interface is compatible with rocSOLVER and cuBLAS-v11 APIs. Porting a CUDA application that originally calls the cuSOLVER API to an application calling the hipSOLVER API should be relatively straightforward. For example, the hipSOLVER SGETRF interface is
+The hipSOLVER interface is compatible with rocSOLVER and cuBLAS-v11 APIs. Porting a CUDA application that originally calls the cuSOLVER API to an application calling the hipSOLVER API should be relatively straightforward (with some caveats; see [here](https://hipsolver.readthedocs.io/en/latest/userguide_intro.html#differences-with-the-cusolver-api)). For example, the hipSOLVER SGEQRF interface is
```

Comment:
If possible, try to ensure the link text is a useful description that can stand on its own. Perhaps, 'see the [List of Differences from the cuSOLVER API](https://hipsolver.readthedocs.io/en/latest/userguide_intro.html#differences-with-the-cusolver-api)'. The article, [What’s wrong with using "Click Here" links?](https://webaccess.berkeley.edu/ask-pecan/click-here) explains why in a bit more detail.

Also, I'm the worst about this sort of thing, but 'relatively straightforward' is weak and vague writing. Relative to what? Drop 'relative' and it's a better sentence.

Reply:
The `relative` part is copied from hipBLAS. I think the statement is too strong with it removed, so perhaps I will replace it with `fairly`.